### ISSUE TITLE
Add i18n to app

### ISSUE THREAD

**Issue by clone1018** [Glimesh/glimesh_app#42]
Right now the app is only available in English. If we can reuse the po files generated for the main website, we can offer i18n pretty easily inside of the app.

Example: https://github.com/Glimesh/glimesh.tv/blob/dev/priv/gettext/pt_br/LC_MESSAGES/default.po

**Comment by ponkey364** [comment]
Looks like there's a library someone wrote for doing this: https://pub.dev/packages/gettext_i18n, been having a play around today and it's working nicely, however I'm not sure how we handle the (few) strings that are in the app and not in the main website or are different between the two?

**Comment by clone1018** [comment]
> however I'm not sure how we handle the (few) strings that are in the app and not in the main website or are different between the two?

I guess I was hoping we could modify all the text in the app to be the same as on the website. This solution wont scale forever, but for now it should work given the parallels between the two platforms.

**Comment by ponkey364** [comment]
yeah, there's a few places locally where I've done that, do you want me to push a branch so you can have a look?

**Comment by clone1018** [comment]
Sounds good to me!

**Comment by ponkey364** [comment]
pushed to the ponkey-test-i18n branch

**Comment by clone1018** [comment]
Looks super straightforward!

**Comment by clone1018** [comment]
@ponkey364 want to turn this into a PR?

**Comment by ponkey364** [comment]
yeah sure, will tidy it up a bit and wrap everything (not just the strings that exist in the PO files) in context.t.
Would it be possible to get the translations synced between POEditor and the glimesh.tv repo so I can make sure we start with an up to date version?

Also, which languages should we include?

(will probably do this at the weekend)

**Comment by clone1018** [comment]
> Would it be possible to get the translations synced between POEditor and the glimesh.tv repo so I can make sure we start with an up to date version?

Wonder if we can submodule the i18n file from Glimesh? I would assume all languages we include on the regular website, but we can always start with English and Spanish if that makes sense to test.

**Comment by clone1018** [comment]
We'll need some kind of selector unless we rely on system language too.

**Comment by ponkey364** [comment]
> Wonder if we can submodule the i18n file from Glimesh? I would assume all languages we include on the regular website, but we can always start with English and Spanish if that makes sense to test.

I'll try the submoduling, I think that's the best idea. I'll set it up to be all the languages we currently support, although we have at least one that Flutter doesn't support, so I'll exclude that.

> We'll need some kind of selector unless we rely on system language too.

Yeah, I was thinking we do both: use the system language, but allow the user to override it in our settings menu?
